### ISSUE TITLE
Add/Implement SAS Procs

### ISSUE THREAD

**Issue by mshin254** [sassoftware/saspy#183]
**Is your feature request related to a problem? Please describe.**
Need more robust econometric methods to implement

**Describe the solution you'd like**
Add methods: autoreg, expand, univariate, varmax, x11, x12

**Comment by mshin254** [comment]
Hi @jld23,
Any update on the implementation?
Thank you.

**Comment by tomweber-sas** [comment]
Jared does the procs for saspy. I'll check with him to see. I know he's been refactoring all of the proc generation code; just opened a PR for all of that. I don't know if these other procs were added as part of that, or if he wanted to get all of that done first, and look into these after that gets merged in.
Thanks,
Tom

**Comment by jld23** [comment]
The PR for refactoring the analytical methods is being reviewed. After that is merged I'll add these procedures.

**Comment by jld23** [comment]
I've just create the PR #192 once it is merged you can try it from master.

**Comment by tomweber-sas** [comment]
And, it's at master now so you can try them out.
Tom

**Comment by jld23** [comment]
Closing. Procs have been pushed.

**Comment by mshin254** [comment]
Hi Jared,

I’m trying to set up the saspy again on a new VDI machine but none of the host/port combinations seem to work.
Anything I do, the following error occurs.
Any insight would be greatly appreciated.
Thank you.

[cid:image001.png@01D6A576.CCC86F20]

Best,

Minchul Shin
Vice President @ Citi
Economic Forecasting Team
P 972-655-3989
E minchul.shin@citi.com
A 6400 Las Colinas Blvd, Irving, TX

**Comment by tomweber-sas** [comment]
Hey @mshin254 , this is a two year old closed issue about adding a proc. Would you mind opening a new issue for this, as it sounds like a configuration problem? That picture didn't come through, can you provide the configuration definition you're trying and the error you're getting? What does VDI mean? And, where is SAS running, and what client machine are you on?
Thanks!
Tom
